assets/xml/textures/icon_item_static.xml

```diff
@@ -1,183 +1,191 @@
+        <Texture Name="gEquipment24Tex" OutName="equipment_2_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gSelectItem04Tex" OutName="select_item_0_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gSelectItem14Tex" OutName="select_item_1_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gSelectItem24Tex" OutName="select_item_2_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gMap04Tex" OutName="map_0_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gMap14Tex" OutName="map_1_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gMap24Tex" OutName="map_2_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gQuestStatus04Tex" OutName="quest_status_0_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gQuestStatus14Tex" OutName="quest_status_1_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gQuestStatus24Tex" OutName="quest_status_2_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gSave04Tex" OutName="save_0_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gSave14Tex" OutName="save_1_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gSave24Tex" OutName="save_2_4" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gGameOver10Tex" OutName="game_over_1_0" Format="ia8" Width="80" Height="32"/>
+        <Texture Name="gABtnSymbolTex" OutName="a_button_symbol" Format="ia8" Width="24" Height="16"/>
+        <Texture Name="gBBtnSymbolTex" OutName="b_button_symbol" Format="ia8" Width="24" Height="16"/>
+        <Texture Name="gCBtnSymbolsTex" OutName="c_button_symbols" Format="ia8" Width="48" Height="16"/>
+        <Texture Name="gNamePanelLeftTex" OutName="name_panel_left" Format="ia8" Width="72" Height="24"/>
+        <Texture Name="gNamePanelRightTex" OutName="name_panel_right" Format="ia8" Width="72" Height="24"/>
+        <Texture Name="gLButtonTex" OutName="l_button" Format="ia8" Width="24" Height="32"/>
+        <Texture Name="gRButtonTex" OutName="r_button" Format="ia8" Width="24" Height="32"/>
+        <Texture Name="gUnknownJPTex" OutName="unknown_JP" Format="ia8" Width="80" Height="16"/>
+        <Texture Name="gSongNoteTex" OutName="song_note" Format="ia8" Width="16" Height="24"/>
+        <Texture Name="gMagicArrowEquipEffectTex" OutName="magic_arrow_equip_effect" Format="ia8" Width="32" Height="32"/>
+        <DList Name="D_020899C0" Offset="0x899C0"/>
+        <DList Name="D_02089A50" Offset="0x89A50"/>
+        <DList Name="D_02089AA0" Offset="0x89AA0"/>
+        <DList Name="D_02089AF0" Offset="0x89AF0"/>
+        <DList Name="D_02089B48" Offset="0x89B48"/>
+        <DList Name="D_02089BA0" Offset="0x89BA0"/>
+        <DList Name="D_02089BF8" Offset="0x89BF8"/>
+        <DList Name="D_02089C48" Offset="0x89C48"/>
+        <DList Name="D_02089C48" Offset="0x89C48"/>
```

Comment:
Another thing I just noticed, shouldn't these be D_08 since this is linked to segment number 8?